caesar_cipher.rb

```diff
@@ -1,25 +1,28 @@
+$lowercase_letters = ('a'..'z').to_a
```

Comment:
I think you can just leave these in the `ceasar_cipher` method if you pass them to the `get_shifted_idx` method.

Reply:
yup yup yup, thank you!